.github/workflows/CI.yml

```
@@ -35,8 +35,7 @@ jobs:
       fail-fast: false
       matrix:
         config:
-          - { runs-on: windows-2019, toolset: v141_xp }
-          - { runs-on: windows-2022, toolset: v143 }
+          - { runs-on: windows-latest, toolset: v143 }
```

```suggestion
          - { runs-on: windows-2022, toolset: v143 }
```
I prefer bumping these manually when they go deprecated so that the environment doesn't change under me